recipes/mofapy/meta.yaml

```diff
@@ -10,20 +10,20 @@ source:
   sha256: 6a5cef1354a1f25a9b9cdf8506d0ee1f59a4e7e6b98a0a59e26705b149b24f7a 
 build:
   noarch: python
-  number: 0
+  number: 1
   script: python -m pip install --no-deps --ignore-installed . 
 
 requirements:
   host:
-    - python
+    - python >=2.7
     - pip
 
   run:
-    - argparse
+    - argparse # [py>27 and py<32]
```

Comment:
argparse has been part of python since 2.7. Since the odds of getting a python2 <2.7 or python3 <3.6 these days is near 0 I suspect this change isn't needed.